app/views/user_mentee_applications/index.html.erb

```diff
@@ -15,12 +15,14 @@
     <% end %>
 
 
-    <h2 class="text-xl font-bold mb-2">Past Applications</h2>
-    <ul>
-      <% @user_mentee_applications.each do |mentee_application| %>
-        <% next if mentee_application.active? && mentee_application.in_review? %>
-        <%= render UserMenteeApplication::ListItemComponent.new(mentee_application:) %>
-      <% end %>
-    </ul>
+    <% if @user_mentee_applications.size.positive? %>
```

Comment:
use `present?` instead of `size.positive?`

Reply:
@afogel - I've addressed your comment. Can you please review?